php/Dockerfile

```diff
@@ -16,5 +16,3 @@ COPY ./conf.d/redis.ini $PHP_INI_DIR/conf.d/redis.ini
 RUN usermod -u 1000 www-data
 
 COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
-
-RUN composer global require hirak/prestissimo
```

Comment:
Since the `composer 2.0` version has been released, it will install dependencies with parallel approach.

And it can be good to remove additional Composer plugin for parallel approach to install dependencies.